office/libreoffice/Portfile

```diff
@@ -62,7 +62,7 @@ checksums           libreoffice-${version}.tar.xz \
                     sha256  f543e6e2d7275557a839a164941c0a86e5f2c3f2a0042bfc434c88c6dde9e140 \
                     size    207992
 
-depends_build-append \
+depends_build \
```

Comment:
Using append is better as its safe against potential future changes, such as adding PG that add dependencies. The is no harm using append if the lists are empty, so I suggest reverting this change.